settings/src/components/totp.scss

```diff
@@ -14,22 +57,58 @@
 	}
 }
 
-.wporg-2fa__manual-code {
-	font-family: monospace;
-	font-weight: bold;
-	letter-spacing: 0.1em;
-}
+.wporg-2fa__manual {
+	display: flex;
+    flex-direction: column;
+    width: fit-content;
+    align-self: center;
+	align-items: center;
 
-.wporg-2fa__verify-code {
-	width: 23ch;
-	margin-bottom: 20px;
+	code {
+		font-family: 'IBM Plex Mono';
+		font-style: normal;
+		font-weight: 400;
+		font-size: 15px;
+		line-height: 20px;
+		margin-top: 16px;
+	}
+	
 }
 
 .wporg-2fa__submit-btn-wrapper {
-	margin-bottom: 15px;
+	width: 100%;
+    display: flex;
+    justify-content: space-between;
+
+	* {
+		flex: 1;
+		justify-content: center;
+	}
+
+	a {
+		background: $white !important;
+		box-shadow: (0 0 0 1px #3858E9) !important;
+		color: #3858E9 !important;
+	}
+
+	button {
+		background: #3858E9 !important;
+		border-color: #3858E9 !important;
+		color: $white !important;
+		margin-right: 0 !important;
```

Comment:
Probably not a good idea to use that many `!important`, but as I don't want to build a button from scratch here, I just overrode the styles of Button from Gutenberg.